Framework/PythonInterface/mantid/api/src/Exports/Sample.cpp

```diff
@@ -35,7 +35,10 @@ void export_Sample() {
       .def("getName", &Sample::getName, return_value_policy<copy_const_reference>(), arg("self"),
            "Returns the string name of the sample")
       .def("getOrientedLattice", (const OrientedLattice &(Sample::*)() const) & Sample::getOrientedLattice, arg("self"),
-           return_value_policy<reference_existing_object>(), "Get the oriented lattice for this sample")
+           return_value_policy<reference_existing_object>(), "Get the first oriented lattice for this sample")
+      .def("getOrientedLattices",
```

Comment:
When I try `ws.sample().getOrientedLattice()` I get this error
```
TypeError: No Python class registered for C++ class class std::vector<class Mantid::Geometry::OrientedLattice,class std::allocator<class Mantid::Geometry::OrientedLattice> >
```

Reply:
Maybe the output has  it has to be converted into a `boost::python::list`? 
Sometimes you have to write a wrapper like this
https://github.com/mantidproject/mantid/blob/573e47ab69796e970592bb03d282ae0c12433206/Framework/PythonInterface/mantid/geometry/src/Exports/PointGroup.cpp#L35-L44

Reply:
Hmm, this should return the first element. I will check it

Reply:
Yes, you are right, I will add this converter